packages/nimble-blazor/NimbleBlazor/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ni/nimble-blazor",
+  "version": "16.0.1",
+  "description": "Blazor components for the NI Nimble Design System",
+  "scripts": {
+    "pack": "cross-env-shell dotnet pack -c Release -p:PackageVersion=$npm_package_version --output ../dist",
```

Comment:
The `pack` script needs to stay in the project `package.json` because it relies on the version number from the file.

Reply:
For both of them lets add the commands used to peerDependencies, i.e. cross-env-shell and rimraf